### ISSUE TITLE
Automatic Resolution and Refres Rate for Game Profiles

### ISSUE THREAD

**Issue by Bagboii** [Valkirie/HandheldCompanion#853]
When playing different games, you prefer to play them with a specific refresh rate to save on battery, or a lower resolution. So under Profiles, it should allow to not change the resolution, or to change it, to something else.

**Comment by CasperH2O** [comment]
Thank you for your request, a resolution or refresh rate Hz setting per profile would be a nice feature.

What do you mean with allow not the change, should HC somehow force or lock the screen resolution after a game changes it?

**Comment by Bagboii** [comment]
> Thank you for your request, a resolution or refresh rate Hz setting per profile would be a nice feature.
> 
> What do you mean with allow not the change, should HC somehow force or lock the screen resolution after a game changes it?

No. The ability to have the profile not have any set resolution, so it stays to what the user had on, or what is default profile.

**Comment by namquang93** [comment]
I second this.

**Comment by GlazedBelmont** [comment]
I third this, would love being able to switch the refresh rate from 60 to 120hz on my plugged/battery profiles
